packages/nvidia_nat_a365/tests/front_end/test_registration.py

```
@@ -0,0 +1,233 @@
+    front_end_types = [fe.config_type for fe in registered_front_ends]
+
+    assert A365FrontEndConfig in front_end_types, (
+        f"A365FrontEndConfig not found in registered front ends. "
+        f"Found: {[fe.config_type.__name__ for fe in registered_front_ends]}"
+    )
+
+
+def test_a365_frontend_config_validation():
+    """Test that A365FrontEndConfig validates required fields."""
+    # Should fail without required fields (app_id is required)
+    with pytest.raises(Exception):  # Pydantic validation error
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)